pkg/controller/remotemachineset/openstackactuator.go

```diff
@@ -0,0 +1,96 @@
+package remotemachineset
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+
+	machineapi "github.com/openshift/cluster-api/pkg/apis/machine/v1beta1"
+	installosp "github.com/openshift/installer/pkg/asset/machines/openstack"
+	installertypes "github.com/openshift/installer/pkg/types"
+	installertypesosp "github.com/openshift/installer/pkg/types/openstack"
+
+	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+)
+
+const (
+	workerUserDataSecretName = "worker-user-data"
```

Comment:
If you want to make this a package-level constant, then (1) it should be moved to a separate file to be used by the other actuators and (2) the other actuators should be changed to use it.

Reply:
It would be nice to do the same thing with `workerRole`, too, so that the OpenStack actuator is not getting the `workerRole` constant from the GCP actuator file.